src/devices/Arduino/README.md

```diff
@@ -0,0 +1,106 @@
+# SPI, GPIO and I2C drivers for Arduino
```

Comment:
you might want to add Firmata somewhere in the name since this will be on the device listing